### ISSUE TITLE
publish "Be Kind to Assholes"

### ISSUE THREAD

**Issue by chadwhitacre** [whit537/whit537.org#24]
@nayafia @hurlothrumbo @kaguillera @jeffspies @clone1018 @jesswhit @dmk246

Hey, folks!

Trying something new here. I've got a piece I'm thinking of publishing, and I'm pinging you for review before doing so. A while back I published "I Resent You" with nary an outside review, and it turned out not to be the bestest of decisions. :wink: I haven't really published under the whit537 banner since then ... I guess I'm wondering if my voice might be about ready to be found again. :no_mouth:

Would you be willing to give this a read, and provide feedback either there on Medium, or here on GitHub, or in email, or in person the next time we see each other? :-) 

**Draft 5**: "**[On the Vilification of 'Assholes'](https://medium.com/@whit537/8ca2ffea91b7)**" (Gratipay's voice)
~~Draft 4: "[On the Vilification of 'Assholes'](https://medium.com/@whit537/41147e46750e)"~~
~~Draft 3: "[On the Vilification of Assholes](https://medium.com/@whit537/24918027cf53)"~~
~~Draft 2: "[On Kindness to Assholes](https://medium.com/@whit537/65fa670ebf83)"~~
~~Draft 1: "[Be Kind to Assholes](https://medium.com/@whit537/2b162efb02f3)"~~

In return, my utmost gratitude. :bow:

chad

**Comment by chadwhitacre** [comment]
P.S. @nayafia I found the presenting tweet from your retweet. Thanks! ;)

**Comment by chadwhitacre** [comment]
Made some changes, Draft 2: "[On Kindness to Assholes](https://medium.com/@whit537/65fa670ebf83)."

"I agree with you," says @JessWhit(!), "but make sure you run it by people actually in the open-source community because the buzzwords you're using don't have the same connotations for me." "What buzzwords?" "'Policing your site.'"

**Comment by chadwhitacre** [comment]
Confabed with @hurlothrumbo this AM. "The first draft was a bit too abrupt, a bit too Andy Rooney:  'Assholes are people, too!' (in an Andy Rooney voice). The second draft is better, but you still need to be clearer that asshole behavior is, in fact, bad. The risk you run here is being seen as condoning assholes."

**Comment by chadwhitacre** [comment]
"This is a bit of a paraphrase, Chad."
"Well, post for yourself."
"I would've just posted a picture of Andy Rooney."

**Comment by chadwhitacre** [comment]
(etc. ;)

**Comment by nayafia** [comment]
Oh my! This is a daring post. ;)

I get the sentiment of what you're saying: don't demonize anyone as "other", try to be inclusive of everyone's perspective instead of writing people off as "assholes". But it feels a little pedantic to me, in that (I assume) the intended sentiment of Mikeal and Anil's posts is "aim to foster a positive, welcoming tone in your community". Which is, I think, actually the same aspiration in the end.

Perhaps including examples of where writing someone off as an "asshole" was harmful, either to the community or the individual? Might help clarify your position.

**Comment by JeffSpies** [comment]
Daring is right....

I was going to second everything else said here and then I read your second draft, which is even more confusing and has even me questioning what you actually mean. Either way, this post seems to be taking a step backwards from what I thought/was hoping for from your brief post on what you learned.

I have a feeling you know what I'm going to say, but I would ask yourself the broader question: is this helping people get paid for open work? 

And then, does gratipay need to be in discussions/commenting on issues like this? If no, then do I need to be in discussions/commenting on issues like this given my role in gratipay? If yes, then I think you have some hard thinking to do.

**Comment by chadwhitacre** [comment]
"What's a better word, vilify or demonize?" "I think vilify."

**Comment by chadwhitacre** [comment]
Draft 3: "[On the Vilification of Assholes](https://medium.com/@whit537/24918027cf53)"

**Comment by chadwhitacre** [comment]
@nayafia @JeffSpies Thanks for weighing in! Just now saw your comments, will digest before proceeding further ...

**Comment by chadwhitacre** [comment]
> Oh my! This is a daring post. ;)
> Daring is right....

Tell me about it! Why am I plagued by such daring thoughts!? :weary: 

I am grateful, however, for the input of friends. Thank you. :-)

**Comment by chadwhitacre** [comment]
> try to be inclusive of everyone's perspective instead of writing people off as "assholes".

@nayafia I wouldn't put it that way. I think it's good and right to exclude perspectives which are racist, sexist, etc. I think we should exclude some people _and_ not call them "assholes." Does that make sense?

> the intended sentiment of Mikeal and Anil's posts is "aim to foster a positive, welcoming tone in your community"

To me, their posts do the opposite. By hating on "assholes" while self-identifying as the "nice people," they foster a negative tone, which I find unwelcoming. I see Mikeal's tweet and the positive response it has garnered and I think, "Gosh, is this what open-source has become? I _really_ do not belong here anymore." I guess I dared to embark on this post because I'm not quite ready to throw in the towel just yet. :-)

> Perhaps including examples of where writing someone off as an "asshole" was harmful, either to the community or the individual? Might help clarify your position.

Good idea! Thanks. :-) I've added a paragraph about this to Draft 4: "[On the Vilification of 'Assholes'](https://medium.com/@whit537/41147e46750e)".

**Comment by chadwhitacre** [comment]
> I was going to second everything else said here and then I read your second draft, which is even more confusing and has even me questioning what you actually mean.

@JeffSpies Sorry. Maybe the [fourth draft](https://medium.com/@whit537/41147e46750e) swings back towards intelligibility?

> Either way, this post seems to be taking a step backwards from what I thought/was hoping for from your brief post on what you learned.

I don't know if Gratipay can be quiescent regarding open-source cultural issues. We're doing some difficult ethical work ([319](https://github.com/gratipay/inside.gratipay.com/issues/319), yes, as well as [here](https://github.com/gratipay/team-review/issues/108) and [here](https://github.com/gratipay/inside.gratipay.com/issues/499)), and I think it might be work that we may as well own.

> I have a feeling you know what I'm going to say, but I would ask yourself the broader question: is this helping people get paid for open work?

You know as well as I, my friend, that money follows vision. "Why is what you're doing the most important thing you could be doing right now?"

> And then, does gratipay need to be in discussions/commenting on issues like this? If no, then do I need to be in discussions/commenting on issues like this given my role in gratipay? If yes, then I think you have some hard thinking to do.

"[On the Vilification of 'Assholes'](https://medium.com/@whit537/41147e46750e)" (Draft 4) is an attempt to say something that I think needs to be said within the open-source subculture. I don't see anyone else saying it, and I think it needs to be said, because hating on "assholes" is sufficiently unhealthy for us to warrant some push-back.

Now, is @whit537 the one to say it? As you suggest, I can't consider my online actions apart from Gratipay. I started this thread thinking that I could start re-engagin as whit537, but because of the entanglement of whit537 with Gratipay, this post probably needs to be something that goes up with an explicit decision from Gratipay (whether under the whit537 or Gratipay name), or not at all.

Hmmm ... how would this read if written in Gratipay's voice? Is ethics Gratipay's marketing plan? Certainly seems like a promising way to differentiate. ;-)

Here's a Draft 5 in Gratipay's voice: "[On the Vilification of 'Assholes'](https://medium.com/@whit537/8ca2ffea91b7)"

Perhaps tomorrow I'll reticket over in inside.gratipay.com ...

**Comment by JeffSpies** [comment]
Some questions:
1. Who is being vilified?
2. What are they being vilified for?
3. Who is vilifying them?

**Comment by chadwhitacre** [comment]
> Some questions:

@JeffSpies Good questions. Happily, though, I've backed away from the edge. It makes no sense for Gratipay to go anywhere near this, and whit537 is so entangled with Gratipay that even if proving someone wrong on the Internet were a good idea for whit537, it'd still be a bad idea here.

> "Why is what you're doing the most important thing you could be doing right now?"

**tl;dr** This isn't. Thank you all for helping me to avoid doing something even stupider than wasting a little bit of our time. :mouse:

[![duty_calls](https://cloud.githubusercontent.com/assets/134455/14093821/abca6e2a-f51e-11e5-8ccd-868ff950520a.png)](https://xkcd.com/386/)
